src/Storage/Part.php

```diff
@@ -294,10 +294,16 @@ public function getHeaders()
      */
     public function getHeader($name, $format = null)
     {
-        $header = $this->getHeaders()->get($name);
+        $headers = $this->getHeaders();
```

Comment:
The API of [`getHeaders`](https://github.com/paragonie-scott/zend-mail/blob/6a53532344db6159cd894082550a40c2a61fb8d1/src/Storage/Part.php#L268) specifies that a `Headers` instance *MUST* be returned. If that is not true, then `getHeaders` needs fixing, and you are fixing a symptom here.